src/Umbraco.Core/Persistence/Constants-Locks.cs

```diff
@@ -80,5 +80,10 @@ public static class Locks
         ///     All webhook logs.
         /// </summary>
         public const int WebhookLogs = -343;
+
+        /// <summary>
+        ///    The cache version.
+        /// </summary>
+        public const int CacheVersion = -344;
```

Comment:
```suggestion
        public const int CacheVersion = -345;
```

Updated just so this doesn't clash with what Laura has proposed in https://github.com/umbraco/Umbraco-CMS/pull/19688